cmd/libvulnhttp/main.go

```diff
@@ -82,7 +82,7 @@ func confToLibvulnOpts(conf Config) *libvuln.Opts {
 		Migrations:               true,
 		UpdaterSets:              nil,
 		DisableBackgroundUpdates: conf.DisableBackgroundUpdates,
-		UpdateInterval:           30 * time.Second,
+		UpdateInterval:           1 * time.Minute,
```

Comment:
libvuln's opt parses defaults anything under 1 minute to the default, 30 minutes. changed this for testing purposes.